bin/faucet/src/config.rs

```diff
@@ -39,8 +34,8 @@ impl FaucetConfig {
 impl Display for FaucetConfig {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         f.write_fmt(format_args!(
-            "{{ endpoint: \"{}\",  database_filepath: {:?}, asset_amount_options: {:?}, token_symbol: {}, decimals: {}, max_supply: {} }}",
-            self.endpoint, self.database_filepath, self.asset_amount_options, self.token_symbol, self.decimals, self.max_supply
+            "{{ endpoint: \"{}\", asset_amount_options: {:?}, token_symbol: {}, decimals: {}, max_supply: {} }}",
+            self.endpoint, self.asset_amount_options, self.token_symbol, self.decimals, self.max_supply
```

Comment:
The database path was used for the client's database so I opted to remove it since it was not needed anymore.

Reply:
This is because we no longer rely on the client here, right?

Reply:
Correct!